index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css"
+   >
+    <link rel="stylesheet" href="style/style.css">
+</head>
+<body>
+    <header>
+        <div class="wrap">
+            <a href ="/">
+                <img class="logo" src="./img/logo/logo.png" width="153">
+            </a>
+            <a href ="/login" class="loginbutton" class="button">로그인</a>
```

Comment:
P1:
class가 두번 기입되었네요. 확인해보세요~
아마 아래가 의도일까요?

```suggestion
            <a href ="/login" class="login-button button">로그인</a>
```